src/main/java/co/elastic/support/diagnostics/commands/RunKibanaQueries.java

```diff
@@ -76,7 +78,7 @@ public int runBasicQueries(RestClient client, DiagnosticContext context) throws
         for (Map.Entry<String, RestEntry> entry : context.elasticRestCalls.entrySet()) {
 
             String actionName = entry.getValue().getName().toString();
-            if (actionName.equals("kibana_alerts") || actionName.equals("kibana_detection_engine_find")) {
+            if (actionName.equals("kibana_alerts") || actionName.equals("kibana_detection_engine_find") || actionName.equals("kibana_security_endpoint_trusted_apps") || actionName.equals("kibana_security_endpoint_exception_host_isolation") || actionName.equals("kibana_security_endpoint_exception_event_filters") || actionName.equals("kibana_security_endpoint_exceotion_trusted_apps") || actionName.equals("kibana_fleet_agents") || actionName.equals("kibana_fleet_agent_policies") || actionName.equals("kibana_fleet_package_policies")) {
```

Comment:
Perhaps for the future, we should add a parameter to the action itself to know if it's paged so that this can become a boolean check instead of this.